assets/js/components/UserMenu.js

```diff
@@ -161,18 +161,19 @@ export default function UserMenu() {
 		<Fragment>
 			<div
 				ref={ menuWrapperRef }
-				className={ classnames(
-					'googlesitekit-user-selector',
-					'googlesitekit-dropdown-menu',
-					'googlesitekit-dropdown-menu__icon-menu',
-					'mdc-menu-surface--anchor',
-					{
-						'googlesitekit-help-menu': unifiedDashboardEnabled,
-					}
-				) }
+				className="googlesitekit-user-selector googlesitekit-dropdown-menu googlesitekit-dropdown-menu__icon-menu mdc-menu-surface--anchor"
 			>
 				<Button
-					className="googlesitekit-header__dropdown mdc-button--dropdown"
+					className={ classnames(
+						'googlesitekit-header__dropdown',
+						'mdc-button--dropdown',
+						// Adds circle background to the non-unified dashboard mobile view
+						'googlesitekit-border-radius-round--phone',
```

Comment:
This leaves out an odd in-between viewport in tablet where it appears as a icon button but still has a rectangular background.